cli/src/potpie/server_manager.py

```
@@ -0,0 +1,371 @@
+                        "--loglevel=info",
+                        "-Q",
+                        f"{celery_queue_name}_process_repository",
+                        "-E",
+                        "--concurrency=1",
+                        "--pool=solo",
+                        ],
+                        stdout=logout,
+                        stderr=logout,
+                    )
+                if(celery_process.returncode != 0):
+                    raise StartServerError(f" Start Celery Failed: {e}")
```

Ruff (0.8.2)

Undefined name `e`

(F821)